### ISSUE TITLE
I'm getting this error when I try to run this code.

### ISSUE THREAD

**Issue by BhavyaTeja** [dheera/android-wearmaps#2]
Information:Gradle tasks [:mobile:generateDebugSources, :mobile:generateDebugAndroidTestSources, :mobile:mockableAndroidJar, :mobile:prepareDebugUnitTestDependencies, :wear:generateDebugSources, :wear:mockableAndroidJar, :wear:prepareDebugUnitTestDependencies, :wear:generateDebugAndroidTestSources]
Error:Execution failed for task ':wear:processDebugResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

**Comment by dheera** [comment]
Google has made a huge number of changes to Android Studio that aren't backward-compatible, unfortunately. Back then I had to cut-and-paste each class one by one each time I upgrade Android Studio between major releases. :-/

(To be honest I don't understand why Gradle needs to be this complicated. 70+ files for a blank app!)

This repo and app are somewhat outdated now that Google Maps has an Android Wear interface -- so I would suggest using that. I wrote this app back in the days when Google Maps was not available on Wear.
